### PR TITLE
lint mesh

### PR DESCRIPTION
**Pull request by almogdepaz**
## Motivation
spring cleaning

## Changes
cosmetic + doc 

## Test Plan
same

### CODE REVIEW

File: mesh/mesh.go [hunk 1]
```diff
@@ -101,13 +110,14 @@ type Mesh struct {
 	txMutex            sync.Mutex
 }
 
-func NewMesh(db *MeshDB, atxDb AtxDB, rewardConfig Config, mesh Tortoise, txInvalidator TxMemPoolInValidator, atxInvalidator AtxMemPoolInValidator, pr TxProcessor, logger log.Log) *Mesh {
+//NewMesh creates a new instant of a mesh
```
Comment (y0sher) [inline mesh/mesh.go:113]:
```suggestion
//NewMesh creates a new instance of a mesh
```

File: mesh/mesh.go [hunk 2]
```diff
@@ -20,71 +20,80 @@ import (
...
+	//Genesis layer index
+	Genesis = types.LayerID(0)
 )
 
-var TRUE = []byte{1}
-var FALSE = []byte{0}
-var LATEST = []byte("latest")
-var LAYERHASH = []byte("layer hash")
-var PROCESSED = []byte("proccessed")
+var constTrue = []byte{1}
```
Comment (y0sher) [inline mesh/mesh.go:28]: how did you come up with `constName` naming ?

**Review by y0sher — APPROVED** [review]

**Comment by almogdepaz** [comment]
bors merge

**Comment by noamnelke** [comment]
Merged as part of **Consolidated linting (#1958)**

**Pull request closed without merging**